src/lib/raft/raft.h

```diff
@@ -236,6 +254,9 @@ struct raft {
 	struct rlist on_update;
 };
 
+static_assert(CHAR_BIT * sizeof(((struct raft *)0)->leader_seen) == VCLOCK_MAX,
+	      "Wrong raft->leader_seen bitmap size");
```

Comment:
You can also use type `vclock_map_t` instead of `uint32_t` probably.

Reply:
Yes, indeed. Fixed. Thanks!